src/App.tsx

```diff
@@ -31,6 +31,17 @@ const { Content } = Layout;
 
 type AppProps = UserAuthenticationReducerState;
 
+export enum ROUTE {
```

Comment:
For consistency, I named this `Routes` on the frontend. Enum values are uppercase, so that's correct. Could you update this to match?